samples/api-client/accesstoken_example/accesstoken.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+Example of using the Google Cloud IoT Core access_token to generate
+gcp compatible token.
+
+Usage example:
+
+    python access.py \\
+      --project_id=my-project-id \\
+      --cloud_region=us-central1 \\
+      --registry_id=my-registry-id \\
+      --device_id=my-device-id \\
+      --certificate_path=/my-certificate.pem \\
+      --scope="scope1 scope2"
+"""
+import argparse
+import io
+import json
+import os
+import time
+from datetime import datetime, timedelta
+
+import jwt
+import requests as req
+
+HOST = "https://staging-cloudiottoken.sandbox.googleapis.com"
+
+
+def generate_gcp_token(
+    project_id, cloud_region, registry_id, device_id, scope, algorithm, certificate_file
+):
+    """Generate GCP access token."""
+    # [START iot_generate_gcp_token]
+    # project_id = 'YOUR_PROJECT_ID'
+    # cloud_region = 'us-central1'
+    # registry_id = 'your-registry-id'
+    # device_id = 'your-device-id'
+    # scope = 'scope1 scope2'
```

Comment:
Add scope list link to comment: https://developers.google.com/identity/protocols/oauth2/scopes